src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `array`, `object`, `arr`, `obj`, `result`. It will decrease code readability. Consider naming the variable in a way that describes what it holds, such as `styleObject` or `cssProperties`.